src/sources/soundsourceffmpeg.cpp

```diff
@@ -474,11 +559,34 @@ bool SoundSourceFFmpeg::readFramesToCache(unsigned int count, SINT offset) {
                     m_SStoredJumpPoint = nullptr;
                 }
 
+#if LIBAVFORMAT_CHANGE_AVSTREAM
+                // AVERROR(EAGAIN) means that we need to feed more
+                // That we can decode Frame or Packet
+                do {
```

Comment:
Ok have to take a look at that have to implement infinite loop checker. Hopefully I have time next week to work on this.